python/paddle/v2/fluid/io.py

```diff
@@ -342,7 +410,13 @@ def load_inference_model(dirname, executor):
         program_desc_str = f.read()
 
     program = Program.parse_from_string(program_desc_str)
-    load_persistables_if_exist(executor, dirname, program)
+    parameter_list = get_parameters(program)
+    load_vars(
```

Comment:
We can also try to call `load_persistables` here.

Reply:
Done